# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: e45d383fec5f
# Full Hash: e45d383fec5fff912b1803b121619b61c6994d62
# Author: Max Leonard Inden <mail@max-inden.de>
# Date: 2025-03-14 21:11:55
# Regressor Bug: 1910360
# File Overlap Count: 1
# Description:
#   Bug 1948739 - enable ECN for QUIC traffic on Beta r=kershaw
#   
#   With the following bugs landed:
#   
#   1. Bug 1910360 enabled quinn-udp on Firefox Nightly.
# ==============================================================================

diff -r 3c4d0d9b6f88 -r e45d383fec5f modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Mar 14 15:55:02 2025 +0200
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Mar 14 12:50:50 2025 +0000
@@ -14594,7 +14594,7 @@
 # true.
 - name: network.http.http3.ecn
   type: RelaxedAtomicBool
-  value: @IS_NIGHTLY_BUILD@
+  value: @IS_EARLY_BETA_OR_EARLIER@
   mirror: always
   rust: true
 
